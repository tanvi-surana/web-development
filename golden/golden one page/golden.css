a
{
	text-decoration: none;
	color: white;
}

#about_main
{
    background-image: url("golden assets/header-bg.jpg");
    color: white;
    padding-bottom: 19%;
}
#services
{
	padding: 40px;
}
/*#contact_us
{
	background-image: url(golden assets/contact-us-bg.jpg)
}*/
#contact_us_heading h1,p
{
	text-align: center;
	/*color: white;*/
}
.major_left
{
	display: flex;
}
.major_right
{
	display: flex;
}
@media (min-width:676px)
{
 #main_nav
 {
        height: 0px;
       display: none;
  }
 #mobile_resize_nav
 {
     display: none;
 }
 #about
 {
    display: flex;
    justify-content: space-between;
 }
 #about ul{
    display: flex;
    list-style: none;
 }
 #about img
 {
    margin-left: 150px;
    margin-top: 10px
    
 }
 #about ul li
 {
    padding-left: 10px;
    padding-right: 15px;
 }

//Services section
	
 #services
{
    display: flex;
    flex-direction: column;
}
#services_margin
{
    padding-left: 10%;
    padding-right: 10%;
}
#services_intro
{
    padding-left: 30%;
}
#services_imgs
{
    display: flex;
}
.service_img
{
    padding-left: 70px;
}
	
/*Team Details*/	

#team_details
{
	display: flex;
	justify-content: space-around;
}

	
#logos
{
	padding-top: 40px;
	padding-bottom: 40px;
	display: flex;
	justify-content: space-around;
}
}

@media (max-width:676px)
{
    #nav_large
    {
        display: none;
    }
    #mobile_resize_nav
    {
        display: block;
    }
    .nav_tool
    {
        width: 22px;
        height: 7px;
        background: black;
        border-radius: 2px;
        border: 1px solid;
    }
    #main_nav
    {
        display: none;
    }
    #main_nav:target
    {
        display: flex;
        flex-direction: column;
        background: rgba(128, 128, 128, 0.78);
    }
	
     /*Change in services on reducing the screen size */
	
#services
{
    display: flex;
  /*  flex-direction:row;*/
}
#services_margin
{
    padding-left: 10%;
    padding-right: 10%;
}
#services_intro
{
    padding-left: 30%;
}
#services_imgs
{
    display: flex;
	flex-direction: column;
}
.service_img
{
    padding-left: 70px;
}
	
	
    
}
#about_para 
{
    /*padding-left: 20%*/
    padding-top: 19%;
}

#about_para #p1
{
    font-size: 30px;
   /* padding-left: 15%;*/
}

#about_para #p2
{
    font-size: 50px;
}

#button_box a
{
	margin-left: 10%;
    text-decoration: none;
    color: white;
}
#about_para div
{
    margin-left: 40%;
    background: rgb(255, 232, 0);
    height: 20px;
    margin-right: 40%;
    padding: 1%;
	border-radius: 7px;
}

 
#portfolio_ul
{
    list-style: none;
    display: flex;
    flex-wrap: wrap;
}
#portfolio_ul li
{
    padding-left: 20px;
}
div.image_size img
{
    width: 300px;
    height: 300px;
}
#portfolio
{
  background: rgb(249, 246, 252);
}
#portfolio_margin
{
     margin-left: 10%;
    margin-right: 10%;
}
#about_circle_margins
{
    margin-left: 30%;
    margin-right: 30%;
}

.circle_img img
{   
    width: 150px;
    height: 150px;
    border-radius: 100px;
    border: 4px solid rgb(249, 246, 252);
}

#team
{
    background: rgb(249, 246, 252);	
	padding-bottom: 30px;
}
#team_margin
{
	padding-left: 10%;
    padding-right: 10%;
}
#team_details p
{
	text-align: center;
}
#team_heading
{
	padding-bottom: 75px;
	padding-top: 50px;
}
#team_heading h1,p
{
	text-align: center;
}


.contact{
    position: relative;
}

.contactcontent{
    position: absolute;
    top:8vw;
    width: 100%;
}
.contactcontent h2{
    color: white;
    text-align: center;
    font-family: arial;
    font-size: 4vw;
    margin-top: 0;
    margin-bottom: 5px;
}

.contactcontent h4{
    color: grey;
    text-align: center;
    font-style: italic;
    font-weight: 100;
    font-family: Noto, Serif,serif;
    font-size: 2vw;
    margin-top: 5px;
    margin-bottom: 5px;
}

.contact input{
    font-size: 2vw;
    font-family: arial;
    border: 1vw solid white; 
    border-radius: 0.5vw;
}

.contact textarea{
    font-size: 2vw;
    font-family: arial;
    border: 1vw solid white; 
    border-radius: 1vw;
    height: 16vw;
    max-height: 200px;
}

.contact form{
    margin-top: 5vw
}
form div{
    display: flex;
    flex-wrap: wrap;
    justify-content: space-around;
}

#input{
    flex-direction: column;
}
.contact .tab{
    background-color: rgb(255,221,73);
    height: 5vw;
    width: 20vw;
    border: 10px solid rgb(255,221,73);
	margin-top: 7vw;
    border-radius: 8px;
    font-size: 2vw;
    display: flex;
}
a{
    text-decoration: none;
}
.contact .tab h4{
    color: white;
    font-weight: 400;
    text-decoration: none;
    margin-top: 1.4vw;
   /* margin-left: 2vw;*/
}

footer {
    margin-top: 20px;
    margin-bottom: 20px;
    display: flex;
    justify-content: space-around;
}

footer h5{
    margin-top: 5px;
    margin-bottom: 5px;
}
footer ul{
    list-style: none;
    display: flex;
}
footer ul li{
    margin-left:2vw;
}
/*.tab
{
	 margin-left: 40%;
    background: rgb(255, 232, 0);
    height: 20px;
    margin-right: 40%;
    padding: 1%;
	border-radius: 7px;
}*/
/*div.line_img
{
    height: 56px;
    width: 2px;
    background: rgb(249, 246, 252);
    margin-left: 224px;
}*/