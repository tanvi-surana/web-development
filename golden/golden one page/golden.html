<!DOCTYPE html>
<html>
    <head>
        <title>Golden!</title>
        <link rel="stylesheet" href="golden.css" type="text/css">
 <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">  
	</head>
    <body>
       <div id="about_main">
           <div id="about">
                <img src="golden assets/main-logo.png" alt="main_logo">  
               
            <a href="#main_nav"><div id="mobile_resize nav">
                   <div class="nav_tool"></div>
                   <div class="nav_tool"></div>
                   <div class="nav_tool"></div>
                   </div>
               </a>
                <nav id="nav_large">
                    <ul>
                        <li><a href="#home">HOME</a></li>
                        <li><a href="#services">SERVICES</a></li>
                        <li><a href="#portfolio">PORTFOLIO</a></li>
                        <li><a href="#about">ABOUT</a></li>
                        <li style="padding-right:100px"><a href="#contact">CONTACT</a></li>
                    </ul>
                </nav>
               <div id="main_nav">
                    <li><a href="#home">HOME</a></li>
                        <li><a href="#services">SERVICES</a></li>
                        <li><a href="#portfolio">PORTFOLIO</a></li>
                        <li><a href="#about">ABOUT</a></li>
                        <li style="padding-right:100px"><a href="#contact">CONTACT</a></li>
               </div>
            </div>
           
           <div id="about_para">
               <p id="p1">Welcome To Our Studio!</p>
               <p id="p2">IT'S NICE TO MEET YOU</p>
               <div id="button_box" style="text-align:center">
               <a href="#">TELL ME MORE</a>
               </div>  
           </div>   
        </div>
        
        
      
        
        
        <div id="services">
           <div id="services_margin">    
            <div id="services_intro">
                <h1>SERVICES</h1>
                <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry</p>
            </div> 
            <div id="services_imgs">
                <div class="services_box">
                    <div class="service_img"><img src="golden assets/e-commerce.png">
                    <h3>E-Commerce</h3>
                    </div>    
                    <div>
                         <p>
                            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
                        </p>
                    </div>
                </div>
                <div class="services_box">
                    <div class="service_img"><img src="golden assets/responsive.png">
                    <h3>Responsive Web</h3>
                    </div>
                    <div>
                         <p>
                            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
                        </p>
                    </div>
                </div>
                <div class="services_box">
                    <div class="service_img"><img src="golden assets/web-security.png">
                        <h3>Web Security</h3>                
                    </div>
                    <div>
                        <p>
                            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
                        </p>
                    </div>
                </div>
            </div>
        </div>
        </div>    

        
<!-- Our Portfolio --> 
        
        
        <div id="portfolio">
            <div id="portfolio_margin">
            <h1>OUR PORTFOLIO</h1>
            <p>acdefghi</p>
            <ul id="portfolio_ul">
                <li>
                    <div class="image_size">
                        <div><img src="golden assets/ebony-ivory.jpg"></div>
                        <div><h4>Ebony and Ivory</h4><p>Branding</p></div>
                    </div>
                </li>
                 <li>
                    <div class="image_size">
                        <div><img src="golden assets/smart-stationary.jpg"></div>
                        <div><h4>Smart Stationary</h4><p>Print Design</p></div>
                    </div>
                </li>
                 <li>
                    <div class="image_size">
                        <div><img src="golden assets/portfolio-3.jpg"></div>
                        <div><h4>Clever Poster</h4><p>Print Design</p></div>
                    </div>
                </li>
                 <li>
                    <div class="image_size">
                        <div><img src="golden assets/portfolio-4.jpg"></div>
                        <div><h4>Vinyl Record</h4><p>Product Mock Up</p></div>
                    </div>
                </li>
                 <li>
                    <div class="image_size">
                        <div><img src="golden assets/portfolio-5.jpg"></div>
                        <div><h4>Treehouse Template</h4><p>Web Design</p></div>
                    </div>
                </li>
                 <li>
                    <div class="image_size">
                        <div><img src="golden%20assets/portfolio-6.jpg"></div>
                        <div><h4>Burned Logo</h4><p>Branding</p></div>
                    </div>
                </li>
                
            </ul>
         </div>    
        </div>
        
        
<!--About-->
        <div id="about">
            <div id="about_circle_margins"> 
              
              <div class="major_left">    
                <div class="circle_img">
                    <div>
                       Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries,  
                    </div>
                    <div>
                    <img src="golden assets/about-us-1.jpg">
                    </div>
                </div>
             </div>
                
                
               <div class="major_right">    
                <div class="circle_img">
                    <div>
                    <img src="golden assets/about-us-2.jpg">
                    </div>
                    <div>
                       Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries,  
                    </div>
                </div>
               </div>    
                
                
                
                <div class="major_left">    
                <div class="circle_img">
                    <div>
                       Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries,  
                    </div>
                    <div>
                    <img src="golden assets/about-us-3.jpg">
                    </div>
                </div>
             </div>
                 
                
                
            
            
               <div class="major_right">    
                <div class="circle_img">
                    <div>
                    <img src="golden assets/about-us-4.jpg">
                    </div>
                    <div>
                       Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries,  
                    </div>
                </div>
               </div>    
                
                
            
            
              <div class="circle_img">
              </div>
        </div>
		</div>
		
		
		
		
	   
<!-- Our Amazing team--> 			
	   <div id="team">
		   <div id="team_margin">
			   <div id="team_heading">
				   <h1>
					   OUR AMAZING TEAM
				   </h1>
				   <p>Lorem Ipsum is simply dummy text of the printing</p>
			   </div>
			  <div id="team_details">
				   <div class="team_members">
					   <div class="team_img">
						   <img src="golden assets/team1.JPG">
					   </div>
					   <div>
						   <h4>Kimberly Thompson</h4>
						   <p>Marketer</p> 
						   <i class="fa fa-twitter"></i>
						   <i class="fa fa-facebook"></i>
						   <i class="fa fa-pinterest-p"></i>
					   </div>
				   </div>
				   
				   <div class="team_members">
					   <div class="team_img">
						   <img src="golden assets/team2.JPG">
					   </div>
					   <div>
						   <p>Kimberly Thompson</p>
						   <p>Marketer</p> 
						   <i class="fa fa-twitter"></i>
						   <i class="fa fa-facebook"></i>
						   <i class="fa fa-pinterest-p"></i>
					   </div>
				   </div>
				   
				   <div class="team_members">
					   <div class="team_img">
						   <img src="golden assets/team3.JPG">
					   </div>
					   <div>
						   <h4>Kimberly Thompson</h4>
						   <p>Marketer</p> 
						   <i class="fa fa-twitter"></i>
						   <i class="fa fa-facebook"></i>
						   <i class="fa fa-pinterest-p"></i>
					   </div>
				   </div>
		    </div>
			   <p>Lorem Ipsum is simply dummy text of the printing</p>
	   </div>
		</div>
	   <div id="logos">
		   <img src="golden assets/logo-1.jpg" >
		   <img src="golden assets/logo-2.jpg" >
		   <img src="golden assets/logo-3.jpg" >
           <img src="golden assets/logo-4.jpg" >
	   </div>
	
		<div id="contact_us">
			<div id="contact_us_heading">
				<h1>CONTACT US</h1>
				<p> Lorem Ipsum is simply dummy text of the printing and</p>
			</div>
		</div>
		
		
		    <div class="contact" id="C">
            <div class="contact-bg">
                <img src="golden assets/contact-us-bg.jpg" width="1400" height="677">
            </div>
            <div class="contactcontent">
                <h2>CONTACT US</h2>
                <h4>Proin iaculis purus consequat sem cure</h4>
            <form>
                <div>
                    <div id="input">
                        <input type="text" name="Name" placeholder="Your Name" size="40"><br/>
                        <input type="text" name="Email" placeholder="E-Mail" size="40"><br/>
                        <input type="text" name="Subject" placeholder="Subject" size="40"><br/>
                    </div>
                        <textarea rows="2" cols="40" placeholder="Message"></textarea>
					
					 <a href="#">
                    <div class="tab">
                        <h4>SEND MESSAGE</h4>
                    </div>
                    </a>
                </div>
				</form>
				</div>
		</div>
				
				
			<footer>
            <div><h5> Copyright2014FreeBeesXpress.com</h5></div>
            <ul>
                <li><img src="google_circle_color-128.png" height="32" width="32"></li>
                <li><img src="circle-facebook_-128.png" height="32" width="32"></li>
                <li><img src="circle-twitter-128.png" height="32" width="32"></li>
                <li><img src="pinterest-128.png" height="32" width="32"></li>
            </ul>
        </footer>
	  <script src="C:\Users\daddy\Desktop\WebD\smoothscroll.js"></script>
    </body>
</html>